### DISCUSSION TITLE
AOSP 12.0 v400.h & Google Pixel XL (marlin)

### DISCUSSION THREAD

**Discussion by SteQuOFFICIAL** [phhusson/treble_experimentations#2159] [General]
Has anyone present here dealt with this system on this device? Can you use all of this in everyday use? Are there any problems?

**Comment by phhusson** [comment]
There is a major issue for everyday use which is that incall audio is
broken.

Le ven. 28 janv. 2022 à 21:52, SteQuOFFICIAL ***@***.***> a
écrit :

**Reply by SteQuOFFICIAL** [reply]
Got it, thank you for the very quick reply. Full professionalism. So I keep my fingers crossed for the chance that this problem will be resolved. I would be very happy. ☺️
